bayesian/api/api_v2.py

```diff
@@ -68,7 +68,8 @@ def get_token():
     try:
         # return default key
         THREESCALE_PREMIUM_USER_KEY = os.getenv('THREESCALE_PREMIUM_USER_KEY')
-        return jsonify({"key": THREESCALE_PREMIUM_USER_KEY}), 200
+        PREMIUM_URL = os.getenv('PREMIUM_URL')
```

Comment:
PREMIUM_URL needs to be added to the template file

Reply:
IT is added @yzainee

Reply:
which template are you talking about?

Reply:
openshift/template.yaml

Reply:
oh i didnt notice.